modules/packages/apk

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh -e
+get_package_data() {
+  name="${INPUT_File?File must be given to get-package-data}"
+  echo PackageType=repo
+  echo Name=$name
+}
+
+list_installed() {
+  # Example pkg output:
+  # Name         Version  Rev   Developer  Notes
```

Comment:
thanks for including this.

Reply:
s/pkg/apk/ and include actual output from `apk` :)